cluster-operator/src/main/java/io/strimzi/operator/cluster/model/Labels.java

```diff
@@ -32,8 +32,8 @@
      *         by the topic operator.</li>
      * </ul>
      */
-    @Deprecated
```

Comment:
You need to fix the java doc, because the kinds are not `(cluster|topic)` any more.